middlewares/accesslog/logger_test.go

```diff
@@ -92,7 +138,7 @@ func printLogdata(logdata []byte) string {
 	return fmt.Sprintf(
```

Comment:
It looks like this part will be executed even in the case where all assertions succeed. Do we want that?